xarray/convert.py

```diff
@@ -157,34 +161,48 @@ def _iris_cell_methods_to_str(cell_methods_obj):
     return ' '.join(cell_methods)
 
 
+def _name(iris_obj, default='unknown'):
+    """ Mimicks `iris_obj.name()` but with different name resolution order.
+
+    Same as iris_obj.name() method, but using iris_obj.var_name first to enable
+    roundtripping.
+    """
+    return iris_obj.var_name or iris_obj.standard_name or \
```

Comment:
Minor nit: per PEP8, please group multi-line statements using an extra set of parentheses instead of using explicit continuation with `\`.